sql/core/src/main/scala/org/apache/spark/sql/execution/metric/CustomMetrics.scala

```diff
@@ -47,7 +47,8 @@ object CustomMetrics {
 }
 
 /**
- * Built-in `CustomMetric` that sums up metric values.
+ * Built-in `CustomMetric` that sums up metric values. Note that please extend this class
+ * and override `name` and `description` to create your custom metric for real usage.
  */
 class CustomSumMetric extends CustomMetric {
```

Comment:
shall we make it an abstract class and force people to implement name and description?

Reply:
okay. had the same thought.